CONTRIBUTING.md

```diff
@@ -39,3 +40,59 @@ To reactivate the virtual environment:
 (host)$ pip install tox
 (host)$ tox
 ```
+
+
+## Backporting commits
+
+For the purpose of these instructions, it is assumed a remote named `upstream` points to this repository and a remote named `fork` points at your fork. We want to backport commits from `devel` to the `release_2.0` branch.
```

Comment:
I'd use `origin` instead of `fork` since that's going to be the default remote name.